api.py

```diff
@@ -9,12 +9,52 @@
 from dotenv import load_dotenv
 from quart_compress import Compress
 import time
+from watchgod import awatch 
+import asyncio
 
 load_dotenv()
 
 app = Quart(__name__)
 Compress(app)
 
+src_dir = "src"
```

Comment:
I would suggest we move this to watchdog.py since this file is named `api.py` so only controllers/quart related things should be here.

Reply:
I have done that. Please review.